tests/outputs_test.go

```diff
@@ -127,3 +128,44 @@ func TestStepLevelAndBundleLevelOutputs(t *testing.T) {
 	err = p.UninstallBundle(uninstallOpts)
 	require.NoError(t, err, "uninstall should have succeeded")
 }
+
+func TestKubernetesMixinOutputs(t *testing.T) {
+	p := porter.NewTestPorter(t)
+	p.SetupIntegrationTest()
+	defer p.CleanupIntegrationTest()
+	p.Debug = false
+
+	// Use the kubernetes example bundle
+	p.CopyDirectory(filepath.Join(p.TestDir, "../examples/kubernetes"), ".", false)
+
+	installOpts := porter.InstallOptions{}
+	installOpts.CredentialIdentifiers = []string{"ci"}
+
+	err := installOpts.Validate([]string{}, p.Porter)
+	require.NoError(p.T(), err, "validation of install opts for bundle failed")
+
+	err = p.InstallBundle(installOpts)
+	require.NoError(p.T(), err, "install of bundle failed")
+
+	// Verify the output value has no wrapping single quotes
+	output, err := p.ReadBundleOutput("IP_ADDRESS", "kubernetes")
+	require.NoError(p.T(), err, "could not read bundle output")
+	require.NotEmpty(p.T(), output, "expected the output value to not be empty")
+	require.NotContains(p.T(), "'", output, "did not expect the output value to contain any single quote characters")
+
+	// Uninstall and delete installation
+	uninstallOptions := porter.UninstallOptions{}
+	uninstallOptions.CredentialIdentifiers = []string{"ci"}
+	uninstallOptions.Delete = true
+
+	err = uninstallOptions.Validate([]string{}, p.Porter)
+	require.NoError(p.T(), err, "validation of uninstall opts for bundle failed")
+
+	err = p.UninstallBundle(uninstallOptions)
+	require.NoError(p.T(), err, "uninstall of bundle failed")
+
+	// Verify that the installation is deleted
```

Comment:
Since we aren't testing that feature here, I don't think we need to assert that uninstall worked as intended. We can just use it and expect it to work. Let the tests that are for that feature do the deeper asserts that check that it is working properly and scope our test to only assert on relevant edge cases for this test (that there aren't the single quotes in the k8s output).